### PR TITLE
Adding hometown file for githubteacher

### PR DESCRIPTION
**Pull request by raghumithinti**
@RamyaYeddula  what kind of restaurant you like so i can add to the hometown file?
@KenTrain1 what kind of restaurant you like so i can add to the hometown file?

### CODE REVIEW

File: raghu_bdc.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ # New Albany, OH
+
+ ### Restaurants
+Cake Walk
+Dosa Corner
```
Comment (hectorsector) [inline raghu_bdc.md]: What kind of food do these restaurants serve? What area are they in?

**Review by hectorsector — CHANGES_REQUESTED** [review]

File: raghu_bdc.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ # New Albany, OH
+
+ ### Restaurants
+Cake Walk
```
Comment (raghumithinti) [inline raghu_bdc.md]: Famous for fresh :cake:

File: raghu_bdc.md (hunk 1 above)
Comment (raghumithinti) [reply raghu_bdc.md]: Freshly prepared :cake: indo-Chinese food and :bread:

**Review by raghumithinti — COMMENTED** [review]
good to try

**Comment by raghumithinti** [comment]
@hectorsector added some restaurants, let me know if it is good :flags:

**Review by hectorsector — APPROVED** [review]
:shipit:

**Pull request merged**